validator.py

```diff
@@ -215,10 +210,26 @@ def objective(k3_trial):
     return diff
 
 
-# In[68]:
+# In[12]:
+
+import scipy.optimize as optimize
+import scipy.cu
+
+
+# In[ ]:
```

Comment:
The preliminary optimization routine of lines 217-242 is not really necessary anymore.